### PR TITLE
write contract step1 UI

### LINKED ISSUE #234: [wallet] contract: P2 2.99 Modal - Write Contract - Step 1

### PR DESCRIPTION
**Pull request by anshuomomo**
**What type of PR is this?**

> /kind ui-change

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #234 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: src/shared/wallet/write-contract-modal.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      <Modal
+        title={<b>{t("wallet.write.contract.title")}</b>}
+        visible={showModal}
+        width={600}
+        bodyStyle={{ paddingBottom: 28, paddingTop: 0 }}
+        footer={null}
+      >
+        <ModalBody>
+          <p>
+            You are about to <b>execute a function on contract.</b>
```
Comment (puncsky) [inline src/shared/wallet/write-contract-modal.tsx]: please add translations. english only.

File: src/shared/wallet/write-contract-modal.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            <br />
+            It will be deployed on the following network: <b>xxxxxxxxxxxx</b>
```
Comment (puncsky) [inline src/shared/wallet/write-contract-modal.tsx]: turn `xxxxxxxxxxxx ` into a prop

File: src/shared/wallet/write-contract-modal.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    }
+  };
+
+  public render(): JSX.Element {
+    const { showModal } = this.props;
+    return (
+      <Modal
+        title={<b>{t("wallet.write.contract.title")}</b>}
+        visible={showModal}
+        width={600}
```
Comment (puncsky) [inline src/shared/wallet/write-contract-modal.tsx]: is it responsive on mobile?

File: translations/zh-CN.yaml [hunk 4]
```diff
@@ -151,3 +151,5 @@ address.balance: 余额
 address.nonce: nonce
 address.pendingNonce: pendingNonce
 button.backHome: 返回首页
+wallet.write.contract.title: 警告
```
Comment (puncsky) [inline translations/zh-CN.yaml]: do not add Chinese translations

File: src/shared/wallet/write-contract-modal.tsx (hunk 2 above)
Comment (anshuomomo) [reply src/shared/wallet/write-contract-modal.tsx]: add address && amount

**Pull request merged**